README.md

```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
 will be found in the parts concerning Rails. But anyways you'll also find a lot about the
 inner workings of Renuo.
 
-* [Ruby on Rails – App Setup Guide](./ruby_on_rails/README.md)
+* [Ruby on Rails – Application Setup Guide](./ruby_on_rails/README.md)
```

Comment:
I want to keep the extended name "Application" everywhere. It's more consistent.